entries/server-entry.js

```diff
@@ -62,10 +75,12 @@ async function reload() {
   const app = await initialize();
   reverseRegister(app, AssetsPlugin);
   reverseRegister(app, ContextPlugin);
+  // $FlowFixMe
```

Comment:
Maybe we should just add this quickly to  https://github.com/fusionjs/fusion-cli/blob/master/flow-typed/globals.js ?